rpcs3/Emu/RSX/Overlays/overlay_edit_text.cpp

```diff
@@ -28,10 +28,90 @@ namespace rsx
 				break;
 			}
 			case up:
+			{
+				if (!caret_position)
+				{
+					// Special case - caret is at the very beginning
+					break;
+				}
+				
+				u64 current_line_start = text.rfind('\n', caret_position - 1) + 1;
+
+				if (current_line_start == std::string::npos)
+				{
+					// This is the first line, so we can't move upwards
+					break;
+				}
+
+				u64 current_line_end = text.find('\n', caret_position + 1);
+
```

Comment:
inconsistent formatting. These three lines are separated but then the next block at line 52 is not. I suggest to just remove the extra newlines here to match the next grouping while saving some vertical space.

Reply:
Also all of them can be const

Reply:
Good point